corehq/apps/reports/forms.py

```diff
@@ -100,36 +101,44 @@ def clean(self):
 
 class ScheduledReportForm(forms.Form):
     config_ids = forms.MultipleChoiceField(
-        label="Saved report(s)",
+        label=_("Saved report(s)"),
         validators=[MinLengthValidator(1)],
         help_text='Note: not all built-in reports support email delivery, so'
                   ' some of your saved reports may not appear in this list')
 
     interval = forms.TypedChoiceField(
-        label='Interval',
+        label=_('Interval'),
         choices=[("daily", "Daily"), ("weekly", "Weekly"), ("monthly", "Monthly")])
 
     day = forms.TypedChoiceField(
-        label="Day",
+        label=_("Day"),
         coerce=int,
         required=False,
         choices=[(i, i) for i in range(0, 32)])
 
     hour = forms.TypedChoiceField(
-        label='Time',
+        label=_('Time'),
         coerce=int,
         choices=ReportNotification.hour_choices())
 
+    start_date = forms.DateField(
+        label=_('Report Start Date'),
+        widget=forms.DateInput(),
```

Comment:
Is this necessary? I'd have guessing `DateField` defaults to `DateInput`

Reply:
Nope, definitely not necessary. Thanks for catching that